scripts/validation/__tests__/validate-labels-before-creation.test.cjs

```
@@ -0,0 +1,251 @@
+      expect(result.warnings.some(w => w.includes('type:bug'))).toBe(true);
+    });
+  });
+
+  describe('Edge Cases', () => {
+    test('handles empty label list', () => {
+      const result = validateLabels([]);
+      expect(result.valid).toBe(false);
+      expect(result.errors.some(e => e.includes("Missing required 'type:*'"))).toBe(true);
+    });
+
+    test('ignores whitespace in labels', () => {
```

ESLint

[error] 'test' is not defined.

(no-undef)